.github/workflows/operator_upgrade.yaml

```diff
@@ -4,16 +4,19 @@ on:
   push:
     branches:
       - main
-      - '*.0.x'
+      - '*.*.x'
```

Comment:
is this ok? iirc this workflow will be triggered only for "push events" targeting 15.0.x branch

Reply:
This is a left over from the commit on the `15.1.x` branch as I had to make the change so that the action was triggered on that branch. I included it here for consistency and to prevent future conflicts when back porting. With the new rule the workflow show be triggered on all branches with the format `*.*.x` and which contain this file workflow file.